apps/frontend/src/emails/templates/moderation/ReportStatusUpdated.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { Heading, Link as VLink, Section, Text } from '@vue-email/components'
+
+import StyledEmail from '@/emails/shared/StyledEmail.vue'
+</script>
+
+<template>
+	<StyledEmail
+		title="Report status updated"
+		:manual-links="[
+			{ link: 'https://modrinth.com/legal/rules', label: 'Community Guidelines' },
+			{
+				link: 'https://modrinth.com/dashboard/report/{report.id}',
+				label: 'View report in dashboard',
+			},
+			{ link: 'https://support.modrinth.com', label: 'Support Portal' },
+		]"
+	>
+		<Heading as="h1" class="mb-2 text-2xl font-bold">Report status updated</Heading>
```

Comment:
should include the name of the item reported, like "Report of '{title}' has been updated" since a lot of people end up reporting a number of projects at once